LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright(c) 2019 Microsoft Corporation

```

Comment:
Is this really the only place the date is found?

Reply:
yes I suppose so, unless you think otherwise?